accumulator_updater/programs/accumulator_updater/src/instructions/put_all.rs

```diff
@@ -26,85 +26,58 @@ pub fn put_all<'info>(
 ) -> Result<()> {
```

Comment:
this method needs some very clear documentation about what it does. Does it overwrite the data in the accumulator input account? What happens if two callers invoke it on the same slot? 

also we had that conversation about being tolerant of an insufficient account size, which also should be documented here. I think callers who want to add new values to an existing list *must do so at the end of the list*

Reply:
just to be clear, adding new values to the end of the list should be the responsibility of the caller but you just want to make sure it's documented here correct?

I added more documentation on `put_all` in `lib.rs`

also i've added an implementation for now which will write as much as possible and then do a log if it wasn't able to write everything.

Reply:
ah I didn't realize the user-facing docs were on the method that calls this. makes sense.